astropy/coordinates/tests/test_frames.py

```diff
@@ -424,6 +424,7 @@ def test_transform():
     i = ICRS(ra=[1, 2]*u.deg, dec=[3, 4]*u.deg)
     f = i.transform_to(FK5)
     i2 = f.transform_to(ICRS)
+    i3 = f.transform_to('icrs')
```

Comment:
This is just a simple smoke test - let me know if there's a better place to put it.